### PR TITLE
Replaced loading svg icon with Loader component

### LINKED ISSUE #15896: Replace the custom SVG icon used for the search loading state with the Loader component
### Problem to solve

Following up from https://github.com/gitpod-io/gitpod/pull/15804, the spinner element used to replace the rotating magnifier icon is a custom SVG. However, we do have a [`Loader`](https://github.com/gitpod-io/gitpod/blob/e940a97c0fe5ab3709a37918a71d07b26a8917a9/components/dashboard/src/components/Loader.tsx) component using an SVG under the hood, which we could replace to use and maintain a consistent icon across the product, see relevant comment in https://github.com/gitpod-io/gitpod/pull/15804#pullrequestreview-1261792254. Cc @andrew-farries @Ciggzy1312 

### Proposal

Replace the custom SVG icon used for the search loading state with the [`Loader`](https://github.com/gitpod-io/gitpod/blob/e940a97c0fe5ab3709a37918a71d07b26a8917a9/components/dashboard/src/components/Loader.tsx) component.

### Risks

Could be worth exploring whether the [`Loader`](https://github.com/gitpod-io/gitpod/blob/e940a97c0fe5ab3709a37918a71d07b26a8917a9/components/dashboard/src/components/Loader.tsx) component is too large to fit into the search input. If that's the case, aligning the size to fit alla cases or introducing two component sizes (small, large) could be the besrt way forward.

### PR DESCRIPTION
**Pull request by Ciggzy1312**
## Description

Replaced the SVG loading icon for search with a new Spinner component
## Related Issue(s)

Fixes #15896 

## Release Notes

```release-note
Replace loading SVG icon with the Loader component
```

## Build Options:

- [ ] /werft with-github-actions
      Experimental feature to run the build with GitHub Actions (and not in Werft).
- [ ] leeway-no-cache
      leeway-target=components:all-ci
- [ ] /werft no-test
      Run Leeway with `--dont-test`

#### Preview Environment Options:
- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

**Comment by roboquat** [comment]
@Ciggzy1312: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by gtsiolis** [comment]
@Ciggzy1312 Could you rebase and resolve conflicts in this one? Thanks! 🙏

**Comment by Ciggzy1312** [comment]
@gtsiolis done, do check it and let me know

File: components/dashboard/src/admin/BlockedRepositories.tsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ import CheckBox from "../components/CheckBox";
 import { ItemFieldContextMenu } from "../components/ItemsList";
 import { ContextMenuEntry } from "../components/ContextMenu";
 import Alert from "../components/Alert";
+import { SpinnerLoader } from "../components/Loader";
```
Comment (gtsiolis) [inline components/dashboard/src/admin/BlockedRepositories.tsx:18]: **issue**: The SVG added seems cropped, although it renders ok. Is this using the code from https://github.com/gitpod-io/gitpod/issues/15896#issuecomment-1400646225? I'm adding below one more version with a square bounding box.

```svg
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 16 16">
  <g clip-path="url(#a)">
    <path fill="#A8A29E" fill-rule="evenodd" d="M11.4284.944629C10.2681.324456 8.97283 0 7.65722 0v3c2.76138 0 4.99998 2.23858 4.99998 5 0 2.7614-2.2386 5-4.99998 5-1.38071 0-2.63071-.5596-3.53553-1.4645l-2.12132 2.1214c.93027.9302 2.07561 1.6167 3.33457 1.9986 1.25896.3819 2.59267.4474 3.883.1908 1.29036-.2567 2.49746-.8276 3.51446-1.6622 1.0169-.8346 1.8124-1.9072 2.3159-3.1226.5034-1.2155.6994-2.53636.5704-3.84564-.129-1.30927-.5788-2.56653-1.3097-3.66042-.7309-1.09389-1.7203-1.99064-2.8806-2.610811Z" clip-rule="evenodd"/>
  </g>
  <defs>
    <clipPath id="a">
      <path fill="#fff" d="M0 0h16v16H0z"/>
    </clipPath>
  </defs>
</svg>
```

**Comment by Ciggzy1312** [comment]
@gtsiolis should be working this time
Let me know if any more changes needed

**Comment by gtsiolis** [comment]
/werft run with-clean-slate-deployment=true with-preview=true recreate-vm=true

**Comment by gtsiolis** [comment]
Thanks, @Ciggzy1312! Updates looks good. ✨ 

@geropl Could you take a look or pass this to someone from the `@gitpod-io/engineering-webapp` team? 🏀

**Comment by gtsiolis** [comment]
Still relevant. Let's loop in @gitpod-io/engineering-webapp again in case this feel between the cracks. 🏓

**Comment by gtsiolis** [comment]
Updated the commits and the SVG icon. 

Removing hold as changes look trivial and this can save the `@gitpod-io/engineering-webapp` from one more review.

/unhold

**Pull request merged**